probe/probe.go

```diff
@@ -291,7 +301,7 @@ func (p *Probe) performGatewayChecks() error {
 	for i := range p.gatewayEndpoints {
 		operationName = "gateway_get_object"
 		s3GatewayTotalCounter.WithLabelValues(operationName, p.name, p.gatewayEndpoints[i].Name).Inc()
-		obj, err := p.gatewayEndpoints[i].s3Client.GetObject(p.gatewayBucketName, objectName, minio.GetObjectOptions{})
+		obj, err := p.gatewayEndpoints[i].s3Client.GetObject(context.Background(), p.gatewayBucketName, objectName, minio.GetObjectOptions{})
```

Comment:
why not using the ctx context instead of the context.Background()?

Reply:
It seems to be the norm in most code example that I saw. It creates an empty context

Reply:
But don't we want to set the context created with the timeout (the ctx one) ?

Reply:
Ah for this one I chose to not timeout because it's not used for latency measurement. I preferred to try as much possible to get the object